### ISSUE TITLE
Docs build results in errors

### ISSUE THREAD

**Issue by fitzage** [joyent/containerpilot#402]
After running `make kirby`, I get the following errors. It's unclear if all the docs get built successfully. Nothing appears to be missing at a glance.

```
bash ./scripts/docs.sh
sed: illegal option -- r
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
sed: 1: "build/docs/10-lifecycle ...": undefined label 'uild/docs/10-lifecycle/docs.md'
sed: illegal option -- r
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
sed: 1: "build/docs/20-design/do ...": undefined label 'uild/docs/20-design/docs.md'
sed: illegal option -- r
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
sed: 1: "build/docs/30-configura ...": undefined label 'uild/docs/30-configuration/31-installation/docs.md'
sed: illegal option -- r
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
sed: 1: "build/docs/30-configura ...": undefined label 'uild/docs/30-configuration/32-configuration-file/docs.md'
sed: illegal option -- r
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
sed: 1: "build/docs/30-configura ...": undefined label 'uild/docs/30-configuration/33-consul/docs.md'
sed: illegal option -- r
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
sed: 1: "build/docs/30-configura ...": undefined label 'uild/docs/30-configuration/34-jobs/docs.md'
sed: illegal option -- r
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
sed: 1: "build/docs/30-configura ...": undefined label 'uild/docs/30-configuration/35-watches/docs.md'
sed: illegal option -- r
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
sed: 1: "build/docs/30-configura ...": undefined label 'uild/docs/30-configuration/36-telemetry/docs.md'
sed: illegal option -- r
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
sed: 1: "build/docs/30-configura ...": undefined label 'uild/docs/30-configuration/37-control-plane/docs.md'
sed: illegal option -- r
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
sed: 1: "build/docs/30-configura ...": undefined label 'uild/docs/30-configuration/38-logging/docs.md'
sed: illegal option -- r
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
sed: 1: "build/docs/30-configura ...": undefined label 'uild/docs/30-configuration/39-examples/docs.md'
sed: illegal option -- r
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
sed: 1: "build/docs/40-support/d ...": undefined label 'uild/docs/40-support/docs.md'
sed: illegal option -- r
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
sed: 1: "build/docs/00-index/doc ...": undefined label 'uild/docs/00-index/docs.md'
```

Generated files:
```
├── 00-index
│   └── docs.md
├── 10-lifecycle
│   └── docs.md
├── 20-design
│   └── docs.md
├── 30-configuration
│   ├── 31-installation
│   │   └── docs.md
│   ├── 32-configuration-file
│   │   └── docs.md
│   ├── 33-consul
│   │   └── docs.md
│   ├── 34-jobs
│   │   └── docs.md
│   ├── 35-watches
│   │   └── docs.md
│   ├── 36-telemetry
│   │   └── docs.md
│   ├── 37-control-plane
│   │   └── docs.md
│   ├── 38-logging
│   │   └── docs.md
│   ├── 39-examples
│   │   └── docs.md
│   └── examples
│       ├── consul-agent.json5
│       ├── database-config.json5
│       ├── nginx-upstreams.json5
│       ├── periodic-tasks.json5
│       ├── service-reg-only.json5
│       └── stopping.json5
└── 40-support
    └── docs.md
```

**Comment by tgross** [comment]
I had GNU sed in my path for various personally quirky reasons and messed that up. Opened https://github.com/joyent/containerpilot/pull/404 with the fix, which I've checked after tossing GNU sed.

**Comment by tgross** [comment]
Fixed in #404
